README.md

```diff
@@ -1,38 +1,97 @@
+</svg>
 
-First, run the development server:
+</div>
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-```
+##_Shoppingify Client_
+
+---
+
+###Overview
+![Imgur](https://i.imgur.com/GounMss.png)
+
+Shoppingify allows you to take your shopping list wherever you go. With Shoppingify, you can easily create personalized shopping lists for any occasion, from weekly grocery shopping to birthday gifts. You can also organize your lists by categories and add useful notes, such as product sizes and preferences. Shoppingify can be accessed on both mobile devices and computers.
+
+---
+
+###Technologies
+:computer: The technologies chosen for this project are:
+
+- <img src="https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png" alt="descrição da imagem" width="13"> Typescript
+- <img src="https://cdn-icons-png.flaticon.com/512/875/875209.png" alt="descrição da imagem" width="13"> React
+- :zap: ChakraUI
+- <img src="https://cdn.worldvectorlogo.com/logos/next-js.svg" alt="descrição da imagem" width="13"> Next
+
+---
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+###How to contribute
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+:memo: If you want to [contribute](https://github.com/joao-pedrozo/shoppingify-client/wiki/How-to-contribute%3F "Contribute") to the project, follow these step-by-step instructions to open a pull request.  
+:question: If you have any questions or issues, you can contact us on [Discord](https://t.co/QWAGh2nd9j).
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+---
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+###How to set up the project
+This is a Next.js project bootstrapped with create-next-app.
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+#####To get started, clone this repository using Git:
 
-## Learn More
+```bash
+# Clone this repository
+$ git clone https://github.com/your-user-name/your-project-name
+```
 
-To learn more about Next.js, take a look at the following resources:
+#####Then, install the project dependencies:
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+```bash
+# Install dependencies
+$ npm install
+```
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+#####Here are the available npm scripts:
```

Comment:
I think that in this beginning of the project we can just tell how to execute the project (`yarn dev`) instead of showing all the available scripts so beginners don't get lost

Reply:
![image](https://user-images.githubusercontent.com/40037644/226758613-962db153-2567-4eed-889c-a159e96a33b3.png)
Here is an example